init-wifi.h

```diff
@@ -149,7 +171,7 @@ bool tryConnectWifi(char* ssid, char* pwd) {
   } else {
     Serial.println();
     Serial.println("Failed to connect to wifi");
-    WiFi.disconnect();
+    //WiFi.disconnect();
```

Comment:
letus remove this or uncomment (do we actually need this?)

Reply:
This line is removed, cause if module didn`t connect to wifi (if condition) - then there is no wifi network to disconnect from.